bagua/torch_api/distributed.py

```diff
@@ -784,6 +828,8 @@ def bagua_init(
         * `module`(_torch.nn.Module_) - Network definition to be run in multi-gpu/distributed mode.
         * `distributed_algorithm`(_DistributedAlgorithm_) - Distributed algorithm used to average
            gradients or weights across all workers. Default: `DistributedAlgorithm.GradientAllReduce`.
+        broadcast_buffers (bool): Flag that enables syncing (broadcasting) buffers of the module 
```

Comment:
The 'broadcast_buffers (bool)' format is the same with pr 'docs'，so i not change other flags here. i wonder if this can be solved by merge in the future.

Reply:
gotcha. thanks